eng/Publishing.props

```diff
@@ -12,10 +12,17 @@
                                        '$(PostBuildSign)' != 'true' and
                                        '$(DotNetBuildRepo)' != 'true'">false</EnableDefaultArtifacts>
 
-    <PublishInstallerBaseVersion Condition="'$(PublishInstallerBaseVersion)' == '' and
-                                            ('$(OS)' == 'Windows_NT' or '$(DotNetBuildOrchestrator)' == 'true')">true</PublishInstallerBaseVersion>
     <!-- This avoids creating VS.*.symbols.nupkg packages that are identical to the original package. -->
     <AutoGenerateSymbolPackages>false</AutoGenerateSymbolPackages>
+    <!--
+      Some assets are produced in all jobs, but only one job can publish them. We follow the following rules in that case:
+      - If we're building outside of the VMR, publish these assets from the Windows job.
+      - If we're building inside the VMR, publish these assets from whichever job is producing non-RID-specific artifacts.
+    -->
+    <PublishAllBuildsAssetsInThisJob Condition="('$(OS)' == 'Windows_NT' and '$(DotNetBuildOrchestrator)' != 'true')
```

Comment:
Have we considered that some npm packages using gyp stuff (native libs) are platform specific? In a long dependency chain, chances of such an npm cannot be ignored.

Reply:
As far as I know, ASP.NET Core has been shipping the npms from their Windows build for years now and there hasn't been an issue. If one arises, then we can address this problem.